src/python/sortingview/gui/pluginInterface/workspaceReducer.ts

```diff
@@ -63,9 +63,18 @@ type SetsnippetLenWorkspaceAction = {
 export type WorkspaceAction = AddRecordingWorkspaceAction | DeleteRecordingsWorkspaceAction | AddSortingsWorkspaceAction | DeleteSortingsWorkspaceAction | DeleteSortingsForRecordingsWorkspaceAction | SetUnitMetricsForSortingWorkspaceAction | SetUserPermissionsAction | SetsnippetLenWorkspaceAction
 
 export const sortingCurationReducer = (state: SortingCuration, action: SortingCurationAction): SortingCuration => {
+    // disable state changes for a closed curation
+    if (action.type !== 'REOPEN_CURATION' && state.isClosed) return state
```

Comment:
Let's print a warning in this case. We should know if this is happening somehow.